rdrf/rdrf/templates/registration/email_reset_completed.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<p>
+  {% trans "Hello" %} {{user_full_name}},<br />
+  {% trans "You're receiving this email because your recent request to change your email address in the" %} {{ registry|title }} {% trans "registry has been completed" %}.<br />
```

Comment:
For current registry `name`s we have:

- `Sample Registry`
- `MiNDAUS Registry`
- `angelman`
- `ARRK`

Which means the word registry appears in the sentence twice for MND and the sample registry. We could reconcile that or change the wording

Reply:
I've changed the wording from "registry" to "site", so:
* the Sample Registry site
* the MiNDAUS Registry site
* the Angelman site
* the ARRK site

What do you think?

Reply:
I also removed the `|title` so that there are no changes to the registry name casing. Angelman might want to look at title casing theirs.